core-test/src/test/scala/org/bitcoins/core/bloom/BloomFilterTest.scala

```diff
@@ -88,7 +88,7 @@ class BloomFilterTest extends BitcoinSUnitTest {
     // for 2 keys
     val numElements = 4
     val bloom = BloomFilter(numElements = numElements,
-                            falsePositiveRate = 0.001,
+                            falsePositiveRate = 0.000001,
```

Comment:
Can we not do 0?

Reply:
I think that would be we would have an infinite size bloom filter. There is a direction relationship between the size of a bloom filter and the rate of false positives allowed. If I have an infinite size bloom filter, and insert 1 element into it, my false positive rate would be 0.000000000000000000000000000000000000001